pkg/monitoring/metrics.go

```diff
@@ -26,10 +27,15 @@ var (
 			Name: "kubevirt_cnao_cr_ready",
 			Help: "Cnao CR Ready",
 		})
+	kmpDeployedGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "kubevirt_cnao_cr_kubemacpool_deployed",
+			Help: "Kubemacpool is deployed by Cnao CR",
+		})
 )
 
 func init() {
-	metrics.Registry.MustRegister(readyGauge)
+	metrics.Registry.MustRegister(readyGauge, kmpDeployedGauge)
```

Comment:
maybe worth that all the gauges will be part of some slice and here we can register with `...` ?
just nice to have

Reply:
I prefer it this way. it's more explicit.